subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -277,6 +277,32 @@ endchoice
 endmenu
 endif
 
+menuconfig BT_MESH_LIGHT_CTRL_REG
+	bool "Lightness PI Regulator"
+	help
+	  Enable the lightness PI regulator module
+
+if BT_MESH_LIGHT_CTRL_REG
```

Comment:
Should coeffs and accuracy configs be moved under this menu?

Reply:
They are up to the user of the reg API, in this case the srv model, so I thought they fit there. Other potential users could or could not have their own such configs.

Reply:
Yes, but they are already in the common regulator structure.

Reply:
Yep, so the API user has to supply them somehow. Either like the server, through config (and set/get messages) or some other method. The regulator does not care.